Comp2024/src/main/java/frc/robot/lib/phoenix/CTREConfigs6.java

```diff
@@ -67,7 +67,7 @@ public static TalonFXConfiguration intakeRotaryFXConfig(double min, double max,
 
     // Slot settings - remote/fused CANcoder affects all feedback constants by the gearRatio
     inRotaryConfig.Slot0.kS = 0.0;                                // Feedforward: Voltage or duty cylce to overcome static friction
-    inRotaryConfig.Slot0.kG = 0.0;                                // Feedforward: Voltage or duty cylce to overcome gravity (arbitrary feedforward) TODO: measure
+    inRotaryConfig.Slot0.kG = 0.5;                                // Feedforward: Voltage or duty cylce to overcome gravity (arbitrary feedforward) TODO: measure
```

Comment:
Since we measured to two decimal places, it would help document this by putting 0.50 instead of 0.5 to let the user know this. Also, please remove the TODO at the end of the line now that it's been measured.